workbench/package.json

```diff
@@ -30,37 +37,32 @@
   "author": "Natural Capital Project",
   "license": "MIT",
   "dependencies": {
-    "bootstrap": "4.3.1",
+    "@babel/runtime": "^7.13.10",
```

Comment:
This was supposed to be here instead of in `devDeps` all along. I think maybe things worked by coincidence because it was included by some other dependencies that were formerly in this section.